app.js

```diff
@@ -36,8 +37,17 @@ client.on('message', async message => {
   }
 })
 
-// console.log(yahooData.getAuthCode().then(resp => console.log(resp)))
-// console.log(yahooData.getInitialToken('').then(resp => console.log(resp)))
-// console.log(yahooData.getRefreshedToken('').then(resp => console.log(resp)))
+// console.log(yahooAuthData.getAuthCode().then(resp => console.log(resp)))
+// console.log(yahooAuthData.getInitialToken('').then(resp => console.log(resp)))
+// console.log(yahooAuthData.getRefreshedToken('').then(resp => console.log(resp)))
 
+console.log(nbaData.getPlayerLeagues(accessTkn)
+  .then(resp => {
+    const leagueId = nbaUtil.getPlayersNbaLeagueId(resp, 'SQA²')
```

Comment:
Was this for testing? It seems to be printing out the leagueId for just the SQA² league. Should remove?

Reply:
yeah the exponent works as expected. Yeah i'll remove this from app.js as it was only for testing.